spring-xd-lang-detector/src/main/resources/config/spring-module.properties

```diff
@@ -0,0 +1,27 @@
+options.textInput.description = The name of the property that contains the input text.
```

Comment:
All descriptions should start with lowercase and NOT end with dot, for consistency